content/en/blog/2023/sig-network-spotlight.md

```diff
@@ -0,0 +1,157 @@
+disparate from each other) upstream Kubernetes efforts are underway to create a
+more modular public interface for `kube-proxy`. The intention is for
+implementations to join in around a common set of libraries and speak a common
+language. This area of focus is known as the KPNG project, and if this sounds
+interesting to you, please join us in the KPNG [community meetings][meet] and
+`#sig-network-kpng` on [Kubernetes Slack][kslack].
+
+[meet]:https://github.com/kubernetes/community/blob/master/sig-network/README.md#meetings
+[kslack]:https://kubernetes.slack.com/
+
+**Multi-Network**
+
+Today one of the primary requirements for Kubernetes networking is to achieve
+connectivity between `Pods` in a cluster, satisfying a large number of
+Kubernetes end-users. However, some use cases require isolated networks and
+special interfaces for performance-oriented needs (e.g. `AF_XDP`, `memif`,
+`SR-IOV`). There's a growing need for special networking configurations in
+Kubernetes in general. The Multi-Network project exists to improve the
+management of multiple different networks for `Pods`: anyone interested in some
+of the lower-level details of `Pod` networking (or anyone having relevant use
+cases) can join us in the Multi-Network community meetings and
+`#sig-network-multi-network` on Kubernetes Slack.
+
+**Network Policy**
+
+The `NetworkPolicy` API sub-group was formed to address network security beyond
+the well-known version 1 of the `NetworkPolicy` resource. We've also been
+working on the `AdminNetworkPolicy` resource (previously known as
+`ClusterNetworkPolicy`) to provide cluster administrator-focused functionality.
+The network policy sub-project is a great place to join in if you're
+particularly interested in security and CNI, please feel free to join our
+community meetings and the `#sig-network-policy-api` channel on Kubernetes
+Slack.
+
+**Gateway API**
+
+If you're specially interested in **ingress** or **mesh** networking the Gateway
+API may be a sub-project you would enjoy. In Gateway API , we're actively
+developing the successor to the illustrious `Ingress` API, which includes a
+`Gateway` resource which defines the addresses and listeners of the gateway and
+various routing types (e.g. `HTTPRoute`, `GRPCRoute`, `TLSRoute`, `TCPRoute`,
+`UDPRoute`, etc.) that attach to `Gateways`. We also have an initiative within
+this project called GAMMA, geared towards using Gateway API resources in a mesh
+network context. There are some up-and-coming side projects within Gateway API
+as well, including `ingress2gateway` which is a tool for compiling existing
+`Ingress` resources to equivalent Gateway API resources and `Blixt`, a Layer4
+implementation of Gateway API using Rust/eBPF for the data plane, intended as a
+testing and reference implementation. If this sounds interesting, we would love
+to have readers join us in our Gateway API community meetings and
+`#sig-network-gateway-api` on Kubernetes Slack.
```

Comment:
Maybe link to the Gateway API website, perhaps also to GAMMA?

Reply:
Added link to Gateway API at the first reference of the it in the paragraph; GAMMA is listed under that so I didn't add it explicitly.